### ISSUE TITLE
Songs in library added with iTunes Match aren't playable

### ISSUE THREAD

**Issue by mikaeleiman** [Musish/Musish#217]
**Describe the bug**
It seems like songs added to my library with iTunes Match aren't playable. I suppose this is by design from Apple's end. Likely only affects songs that aren't actually matched with something that's available on Apple Music.

**To Reproduce**
1. Add a song to iTunes and have iTunes Match 
2. Try playing the song

**Comment by mikaeleiman** [comment]
app.f202ca1d.js:48:17249: 
Unhandled promise rejection UNKNOWN_ERROR: "UNKNOWN_ERROR"
	MKError https://js-cdn.music.apple.com/musickit/v1/musickit.js:1:24902
	serverError https://js-cdn.music.apple.com/musickit/v1/musickit.js:1:25749
	_handleLicenseResponse https://js-cdn.music.apple.com/musickit/v1/musickit.js:1:103813
	startLicenseSession https://js-cdn.music.apple.com/musickit/v1/musickit.js:1:101998
	a https://musi.sh/app.f202ca1d.js:48:16869
	M https://musi.sh/app.f202ca1d.js:48:17014
	s https://musi.sh/app.f202ca1d.js:48:404876

**Comment by eriksf** [comment]
Yes, I'm also seeing these errors with purchased music that is not available in Apple Music. In one case (Heavenly Father - Bon Iver - Wish I Was Here (Music From the Motion Picture)), the track is still available in the iTunes store, and in the other case (From What I Once Was - Neverending White Lights - Act 1: Goodbye Friends of the Heavenly Bodies) it's no longer available in the store. Either way, MusicKit should be able to handle this (iTunes or the iOS Music player do).

**Comment by Kail0** [comment]
Even songs uploaded to iCloud library via subscription to Apple Music (it's basicaly same as the iTunes Match) are not working. Just FYI.
It would be nice to play them, tho.
